WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -30,22 +33,25 @@
 #import "BlogDetailHeaderView.h"
 #import "ReachabilityUtils.h"
 
-const typedef enum {
+typedef NS_ENUM(NSInteger, BlogDetailsRow) {
     BlogDetailsRowViewSite = 0,
```

Comment:
I think that'd be more confusing. If we're being nitpicky, I don't think this should be an `enum` at all, just independent constants :wink: